source/style-guide/writing/write-to-the-user.txt

```diff
@@ -22,26 +28,49 @@ talk to users directly by using *second person*, addressing the user as
       voice.
 
 -  Use second person to avoid the use of gender-specific, third-person
-   pronouns such as *he*, *she*, *his*, and *hers*. If you must use
-   third person, use the pronouns *they* and *their*, but ensure that
-   the noun that the pronoun refers to is plural.
+   pronouns:
+
+   .. hlist::
+      :columns: 4
+
+      - *he*
+      - *she*
+      - *his*
+      - *hers*
+      - *him*
+      - *her*
+      - *himself*
+      - *herself*
+
+   If you must use third person, use the pronouns:
+
+   .. hlist::
+      :columns: 4
+
+      - *they*
+      - *them*
+      - *their*
+      - *themselves*
+
+   Verify that the pronoun's antecedent is plural.
```

Comment:
Recommend additional callout:

Situations where third person makes sense are rare. Second person should address the vast majority of your documentation needs when writing reference and instructional material.

Reply:
Added a modified version.